File: lab 7.java
import java.util.Scanner;

class Student {
    String name;
    int rollNumber;
    float marks;

    void inputDetails() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter student name: ");
        name = sc.nextLine();
        System.out.print("Enter roll number: ");
        rollNumber = sc.nextInt();
        System.out.print("Enter marks: ");
        marks = sc.nextFloat();
    }

    void displayDetails() {
        System.out.println("\nStudent Details:");
        System.out.println("Name: " + name);
        System.out.println("Roll Number: " + rollNumber);
        System.out.println("Marks: " + marks);
    }
}

public class Main {
    public static void main(String[] args) {
        Student student = new Student();
        student.inputDetails();
        student.displayDetails();
    }
}

File: lab 7.cpp
7. Write a program to display student details using class.( By C++ or Java )


#include <iostream>
using namespace std;

class Student {
public:
    string name;
    int rollNumber;
    float marks;

    void inputDetails() {
        cout << "Enter student name: ";
        getline(cin, name);
        cout << "Enter roll number: ";
        cin >> rollNumber;
        cout << "Enter marks: ";
        cin >> marks;
    }

    void displayDetails() {
        cout << "\nStudent Details:\n";
        cout << "Name: " << name << endl;
        cout << "Roll Number: " << rollNumber << endl;
        cout << "Marks: " << marks << endl;
    }
};

int main() {
    Student student;
    cin.ignore();
    student.inputDetails();
    student.displayDetails();
    return 0;
}
